sgl-kernel/tests/test_sampling_scaling_penalties.py

```diff
@@ -29,7 +33,3 @@ def test_sampling_scaling_penalties(batch_size, vocab_size, dtype):
         atol=atol,
         msg=f"Failed for batch_size={batch_size}, vocab_size={vocab_size}, dtype={dtype}",
     )
-
-
-if __name__ == "__main__":
```

Comment:
why remove this?

Reply:
To unify the style with other pytest tests.

Reply:
I think other tests use it too.

Reply:
ref https://github.com/sgl-project/sglang/blob/665e5e85f6d7a3a153d852cf11f73ba2f892fdff/sgl-kernel/tests/test_norm.py#L132-L133

Reply:
Okay, I'll add it back when I'm free later.